<launch>
  <arg name="nao_ip" default="$(optenv NAO_IP 10.0.1.236)" />
  <arg name="nao_port" default="$(optenv NAO_PORT 9559)" />
  
  <node pkg="group3" type="wakeup_node.py" name="wakeup_node" required="true" args="--ip=$(arg nao_ip) --port=$(arg nao_port)" output="screen" />
  <node pkg="group3" type="head_motion_service_server.py" name="head_motion_service_server" required="true" args="--ip=$(arg nao_ip) --port=$(arg nao_port)" output="screen" />
  <node pkg="group3" type="text2speech_node.py" name="text2speech" required="true" args="--ip=$(arg nao_ip) --port=$(arg nao_port)" output="screen" />
  <node pkg="group3" type="arm_controller_service_server.py" name="arm_controller_service_server" required="true" args="--ip=$(arg nao_ip) --port=$(arg nao_port)" output="screen" />
  <node pkg="group3" type="detector_object.py" name="detector_object" required="true" args="--ip=$(arg nao_ip) --port=$(arg nao_port)" output="screen" />


</launch>
